ksql-engine/src/test/java/io/confluent/ksql/integration/JoinIntTest.java

```diff
@@ -107,10 +126,50 @@ public void shouldLeftJoinOrderAndItems() throws Exception {
     }, 60000, "failed to complete join correctly");
   }
 
+  @Test
+  public void shouldLeftJoinOrderAndItemsJson() throws Exception {
+    shouldLeftJoinOrderAndItems("ORDERWITHDESCRIPTIONJSON",
+                                orderStreamTopicJson,
+                                orderStreamNameJson,
+                                itemTableNameJson,
+                                DataSource.DataSourceSerDe.JSON);
+
+  }
+
+  @Test
+  public void shouldLeftJoinOrderAndItemsAvro() throws Exception {
+    shouldLeftJoinOrderAndItems("ORDERWITHDESCRIPTIONAVRO",
+                                orderStreamTopicAvro,
+                                orderStreamNameAvro,
+                                itemTableNameAvro,
+                                DataSource.DataSourceSerDe.AVRO);
+  }
+
+
   private void createStreams() throws Exception {
-    ksqlContext.sql("CREATE STREAM orders (ORDERTIME bigint, ORDERID varchar, ITEMID varchar, ORDERUNITS double, PRICEARRAY array<double>, KEYVALUEMAP map<varchar, double>) WITH (kafka_topic='" + orderStreamTopic + "', value_format='JSON');");
-    ksqlContext.sql("CREATE TABLE items (ID varchar, DESCRIPTION varchar) WITH (kafka_topic='" +
-                    itemTableTopic + "', value_format='JSON', key='ID');");
+    ksqlContext.sql(String.format("CREATE STREAM %s (ORDERTIME bigint, ORDERID varchar, ITEMID "
```

Comment:
Can we get the format of each statement such that it is consistent within this method?